src/Microsoft.AspNetCore.Localization.Routing/Microsoft.AspNetCore.Localization.Routing.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We want to avoid dependency of Routing in core localization package as it does not depend on routing.